.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # Beacon for IBM.com team
-* @IBM/beacon-for-ibm-com-members
+* @IBM/beacon-members
```

Comment:
@jeffchew Not sure if this is the correct change. Is it `<repo-name>-members`?

Reply:
we would have to change the name of the team too, let's leave this one alone for now